src/character.rs

```diff
@@ -0,0 +1,88 @@
+use crate::animation::Animation;
+use bevy::prelude::*;
+use bevy_rapier2d::prelude::*;
+use std::time::Duration;
+
+#[derive(Component, Copy, Clone)]
+pub enum Direction {
+	Right,
+	Left,
+}
+
+pub struct DefaultCharacterConfig {
+	starting_x: f32,
+	starting_y: f32,
+	sprite_render_width: f32,
+	sprite_render_height: f32,
+	cycle_delay: Duration,
+	velocity: Velocity,
+	direction: Direction,
+}
+
+#[derive(Component, Copy, Clone)]
+pub struct Velocity {
+	pub x: f32,
+	pub y: f32,
+}
+
+pub static DEFAULT_CHARACTER_CONFIG: DefaultCharacterConfig =
+	DefaultCharacterConfig {
+		starting_x: 0.0,
+		starting_y: 0.0,
+		sprite_render_width: 64.0,
+		sprite_render_height: 64.0,
+		cycle_delay: Duration::from_millis(100),
+		velocity: Velocity { x: 300.0, y: 650.0 },
+		direction: Direction::Right,
+	};
```

Comment:
Nit: the `Velocity` struct seems a bit lost between the declaration and then instantiation of the `DefaultCharacterConfig` struct... Maybe move it below or above? But it's just a nit

Reply:
Agree! will move this then merge in